comit/src/network/protocols/orderbook.rs

```diff
@@ -27,7 +31,7 @@ use std::{
     time::Duration,
 };
 
-pub use self::order::*;
+pub use self::{order::*, orders::*, quote::*};
 
 /// String representing the BTC/DAI trading pair.
 const BTC_DAI: &str = "BTC/DAI";
```

Comment:
This const is used for topic creation, and BTC/DAI orders are published under that topic ... I'll have a look, I don't think it will be nice Rust but its a fair comment, thanks.